pkgs/development/python-modules/lxml-html-clean/default.nix

```diff
@@ -26,15 +23,31 @@ buildPythonPackage rec {
 
   dependencies = [ lxml ];
 
-  nativeCheckInputs = [ unittestCheckHook ];
+  nativeCheckInputs = [
+    # Tests should be run with unittest, which pytest is not completely compatible with.
+    # However, only pytest allows us to skip specific tests.
+    pytestCheckHook
+  ];
```

Comment:
IMO, this would be cleaner as a patch to the original source code to remove/comment out the tests. That way we could keep using the correct test runner.

(I don't feel strongly about this, feel free to ignore this)

Reply:
This is fair. Done!